datafusion/datasource-parquet/src/row_filter.rs

```diff
@@ -803,9 +689,10 @@ mod test {
     fn basic_expr_doesnt_prevent_pushdown() {
         let table_schema = get_basic_table_schema();
 
-        let file_schema = Schema::new(vec![Field::new("str_col", DataType::Utf8, true)]);
+        let file_schema =
+            Schema::new(vec![Field::new("string_col", DataType::Utf8, true)]);
```

Comment:
There is no `str_col` in the data returned by `get_basic_table_schema()` but there is `string_col`.